components/apps.js

```diff
@@ -545,37 +561,28 @@ SteamUser.prototype._getLicenseInfo = async function() {
 /**
  * Get list of appids this account owns. Only works if enablePicsCache option is enabled and appOwnershipCached event
  * has been emitted.
- * @param {boolean} [excludeSharedLicenses=false] - Pass true to exclude licenses that we have through family sharing
+ * @param {object} options - Options for what counts for ownership
```

Comment:
Parameter `excludeSharedLicenses` has now become `options`. For now, I minimized it to the essential. So, it only supports the following options:
 * `options.shared` - Include shared licenses (from steam family sharing)? Default: false.
 * `options.expiring` - Include expiring licenses (free weekends)? Default: false.
 
 There is still support for the old parameter (implemented [here](https://github.com/DoctorMcKay/node-steam-user/pull/352/files#diff-e4f6f60b0179df92e9cb07d64cf7eb12808425212d81e90ae5f410e267a915a0R677)).

Reply:
Maybe I could also add a `free` filter that filters out free on demand and noCost packages and sub 0 too. I think it should still include free promotions though, since those are given away for you to keep.